types/livereload/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Hector Osuna <https://github.com/FanGoH/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import { Server as WebSocketServer } from 'ws';
+import { Server as WebSocketServer, WebSocket } from 'ws';
```

Comment:
Sorry, I'm also confused about what problem this fixes.  The tests pass for me after adding the export below.

Reply:
The test is falsepositive then. When I used this package, I got WebSocket type was undefined error.